helm-chart-sources/pulsar/templates/broker-deployment/broker-configmap.yaml

```diff
@@ -71,10 +71,10 @@ data:
   {{- end }}
   {{- if .Values.broker.kafkaOnPulsarEnabled }}
   # enabling Kafka-on-Pulsar
-  protocolHandlerDirectory: "./protocols"
+  proxyExtensionsDirectory: "./extensions"
```

Comment:
on the Broker we still have "protocol handlers"
you have to change only the parts related to "proxy"